app/views/movies/_movie.html.erb

```diff
@@ -7,13 +7,13 @@
   <table class="table table-hover mx-auto">
   <thead>
     <tr>
-      <th scope="col">Seances times</th>
+      <th scope="col">Seances for next week</th>
     </tr>
   </thead>
   <tbody>
     <% @movieseances.each do |seance| %>
```

Comment:
This is not part of your PR but this is a bad variable name
Should either be `@movie_seances` or just `@seances`